plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridTableColumnManager.java

```diff
@@ -44,6 +44,18 @@
 
 public class CubridTableColumnManager extends GenericTableColumnManager implements DBEObjectRenamer<GenericTableColumn>
 {
+   public ColumnModifier<GenericTableColumn> CubridDataTypeModifier = (monitor, column, sql, command) -> {
+        final String typeName = column.getTypeName();
+        DBPDataKind dataKind = column.getDataKind();
+        sql.append(' ').append(typeName);
+        String modifiers = SQLUtils.getColumnTypeModifiers(column.getDataSource(), column, typeName, dataKind);
+        if (modifiers != null && !typeName.equals("STRING")) {

```

Comment:
I suggest using `equalsIgnoreCase` instead, same below.

Reply:
Thanks for your suggestion, I will update it.